ecosystem/typescript/sdk_v2/package.json

```diff
@@ -29,29 +29,30 @@
   },
   "dependencies": {
     "@aptos-labs/aptos-client": "^0.0.2",
+    "@noble/curves": "^1.2.0",
     "@noble/hashes": "1.1.3",
```

Comment:
do we also need `@noble/hashes`?

Reply:
Looks like not being added by this repo. Maybe its needed by other stuff?